test/rlcIntegration/generated/lroRest/package.json

```diff
@@ -82,11 +82,5 @@
     "rollup": "^2.66.1",
     "rollup-plugin-sourcemaps": "^0.6.3",
     "uglify-js": "^3.4.9"
-  },
-  "//sampleConfiguration": {
```

Comment:
Was this generated before ?

Reply:
Yes, it is introduced by the first time I init the sample option with a prototype sample file. This time I disable this option for LRORest.